staging/src/k8s.io/cloud-provider/cloud.go

```diff
@@ -199,9 +199,10 @@ type Routes interface {
 }
 
 var (
-	InstanceNotFound = errors.New("instance not found")
-	DiskNotFound     = errors.New("disk is not found")
-	NotImplemented   = errors.New("unimplemented")
+	DiskNotFound         = errors.New("disk is not found")
+	ImplementedElsewhere = errors.New("Implemented by a different controller")
```

Comment:
Please do not capitalize the first letter of an error.

Reply:
done